frontend/src/pages/RunDetails.tsx

```diff
@@ -853,19 +864,36 @@ class RunDetails extends Page<RunDetailsInternalProps, RunDetailsState> {
   private async _loadSidePaneTab(tab: SidePaneTab): Promise<void> {
     const workflow = this.state.workflow;
     const selectedNodeDetails = this.state.selectedNodeDetails;
+
+    let sidepanelBannerMode = 'warning' as Mode;
+
     if (workflow && workflow.status && workflow.status.nodes && selectedNodeDetails) {
       const node = workflow.status.nodes[selectedNodeDetails.id];
       if (node) {
         selectedNodeDetails.phaseMessage =
           node && node.message
             ? `This step is in ${node.phase} state with this message: ` + node.message
             : undefined;
+
+        selectedNodeDetails.phase = node.phase;
+
+        switch (node.phase) {
+          case NodePhase.ERROR:
+            sidepanelBannerMode = 'error';
+            break;
+          case NodePhase.FAILED:
+            sidepanelBannerMode = 'warning';
```

Comment:
Sorry I don't have much context here, can you help me understand the difference between `ERROR` and `FAILED`?

Reply:
I'll be honest I'm not 100% of this either, but I would understand this to be 
- ERROR if there was a error trying to set up the container or something, or network error, or issue in argo, and
- FAILED means the _execution_ of the container failed

Reply:
Thanks for the explanation, I'm not sure I understand why you want to show `warning` for failed node.
For users, it seems to me they should also perceive it as an error.

Reply:
It's to try to separate pipelines failure from system failure, but happy to change to error type too

Reply:
Understood. IMO, can we add some wording in the error message to indicate difference between `fail` and `error`.
e.g. `error` -> `Pipelines System Error: original message`; while `fail` -> `Failed execution: ...`

what do you think?

This might be a little controversial, we can also revert this part keeping current behavior, so that we can get this PR merged. Opening a new PR for this specific issue.

Reply:
Well I'll make em all Error type, and we can add a separate issue to add those prefixes?

Reply:
Sounds good! Thank you!